docs/meshcloud.policies.md

```diff
@@ -5,8 +5,18 @@ title: meshPolicies
 
 ## Introduction: what are meshPolicies?
 
-A meshPolicy is a set of rule(s) between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each rule describes what [tag](meshcloud.tag-schema.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects.
-  
+A meshPolicy is a set of rule(s) between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each rule describes what [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+
+### meshPolicies for meshUsers/Groups
+
+meshUsers and meshCustomerUserGroups are treated as one when it comes to meshPolicies. You can only define a policy for "meshUser/Group". Those policies will always apply to both meshObject types. It is not possible to define a policy that only matches meshUsers, but not meshCustomerUserGroups. The reason for that is, that you can assign both meshObject types (meshUsers and meshCustomerUserGroups) to meshCustomers and meshProjects. If you want to restrict this access to e.g. only allow access to production for certains users and groups, the policy always has to apply to both meshObject types. It wouldn't make sense to restrict only the assignment of groups, but you could still assign any user. Because of that, you can only select "meshUser/Group" as a policy subject in a meshPolicy.
+
+meshUsers exist on a global level and are not related to a specific meshCustomer. If you want to define a meshPolicy to only provide certain meshUsers/Groups access to a e.g. production projects, the following aspects have to be considered:
+
+1. If a user is globally defined to have access to any production project, you can add the production environment tag to that user, so this user gets access to all production projects in all meshCustomers.
```

Comment:
This is not possible right? All users have the same tags?

Reply:
actually this is now possible as i added the option to define tags on meshUsers via meshObject import. The important thing is, that it is globally defined, so it affects all customers. That's what i described here. This doesn't match the CoBa usecase, but it is still a valid usecase in other scenarios imo.

Reply:
I'm not questioning if it's good or not but just if it exists, I wasn't aware this is now possible so forget my comment ;-)

Reply:
actually i was questioning myself whether it is good to actually provide setting tags on users via meshObject API, but in the end i liked it and i needed for some scenario tests :P